src/components/Footer.astro

```diff
@@ -1,88 +1,49 @@
-
-
-<section class="border border-solid border-black w-full min-h-96">
-    <!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
-    <title>Footer</title>
-    <style>
-        body {
-            background-color: #000;
-            margin: 80px 60px 0px 60px;
-        }
-    </style>
-</head>
-
-<body class="text-black">
-
-    <footer class="py-8 mx-4 my-4">
-
-        <!-- Logo and Contact -->
-        <div class="flex justify-between items-center mb-4">
-            <!-- Logo -->
-            <div class="mr-4">
-                <img src="path/to/your/logo.png" alt="Logo" class="h-8">
-            </div>
-
-            <!-- Contact -->
-            <div class="text-blue-500">
-                <p>Contact</p>
-            </div>
-        </div>
-
-        <!-- Mission Statement and Address -->
-        <div class="flex justify-between mb-4">
-            <!-- Mission Statement -->
-            <div>
+<footer class="border border-solid border-black w-full min-h-96 p-4">
+    <div class="container mx-auto m-16">
+        <div class="flex justify-between items-center">
+            <!-- Left Container: Logo, Mission Statement, and Social Media Icons -->
+            <div class="flex flex-col space-y-6 items-start ml-6"> 
+                <img src="/public/poster.svg" alt="Logo" class="h-12"> 
                 <p class="text-gray-300">Tech. Design. Reimagine.</p>
+                <div class="flex space-x-2">
+                    <a href="#" class="text-white">
+                        <img src="/public/instagram.svg" alt="Instagram" class="h-9">
```

Comment:
Please update the image `src` paths by removing the `/public` prefix; it's unnecessary. Use relative paths like `src="/instagram.svg"` instead.